src/Sulu/Component/Content/Tests/Functional/Rlp/Mapper/PhpcrMapperTest.php

```diff
@@ -240,174 +294,100 @@ public function testMoveTwice()
-        $this->rlpMapper->move('/news', '/neuigkeiten', 'sulu_io', 'de');
-    }
-
-    public function testMoveTree()
-    {
-        $session = $this->sessionManager->getSession();
-
-        // create routes for content
-        $this->rlpMapper->save($this->content1, '/news', 'sulu_io', 'de');
-        $this->rlpMapper->save($this->content1, '/news/news-1', 'sulu_io', 'de');
-        $this->rlpMapper->save($this->content1, '/news/news-1/sub-1', 'sulu_io', 'de');
-        $this->rlpMapper->save($this->content1, '/news/news-1/sub-2', 'sulu_io', 'de');
-
-        $this->rlpMapper->save($this->content1, '/news/news-2', 'sulu_io', 'de');
-        $this->rlpMapper->save($this->content1, '/news/news-2/sub-1', 'sulu_io', 'de');
-        $this->rlpMapper->save($this->content1, '/news/news-2/sub-2', 'sulu_io', 'de');
-        $session->save();
-
-        // move route
-        $this->rlpMapper->move('/news', '/test', 'sulu_io', 'de');
-        $session->save();
-        $session->refresh(false);
-
-        // check exist new routes
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test', 'sulu_io', 'de')
-        );
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-1', 'sulu_io', 'de')
-        );
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-1/sub-1', 'sulu_io', 'de')
-        );
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-1/sub-2', 'sulu_io', 'de')
-        );
-
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-2', 'sulu_io', 'de')
-        );
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-2/sub-1', 'sulu_io', 'de')
-        );
-        $this->assertEquals(
-            $this->content1->getIdentifier(),
-            $this->rlpMapper->loadByResourceLocator('/test/news-2/sub-2', 'sulu_io', 'de')
-        );
-
-        // check history
-        $this->assertEquals('/test', $this->getRlForHistory('/news'));
-        $this->assertEquals('/test/news-1', $this->getRlForHistory('/news/news-1'));
-        $this->assertEquals('/test/news-1/sub-1', $this->getRlForHistory('/news/news-1/sub-1'));
-        $this->assertEquals('/test/news-1/sub-2', $this->getRlForHistory('/news/news-1/sub-2'));
-
-        $this->assertEquals('/test/news-2', $this->getRlForHistory('/news/news-2'));
-        $this->assertEquals('/test/news-2/sub-1', $this->getRlForHistory('/news/news-2/sub-1'));
-        $this->assertEquals('/test/news-2/sub-2', $this->getRlForHistory('/news/news-2/sub-2'));
-    }
```

Comment:
This test has been moved to the new `TreeStrategyTest`, because the `PhpcrMapper` is not handling children anymore.